src/openforms/contrib/kadaster/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+from django.utils.translation import gettext_lazy as _
+
+
+class KadasterAPIConfig(AppConfig):
+    name = "openforms.contrib.kadaster"
+    label = "contrib_kadaster"
+    verbose_name = _("Kadaster API")
```

Comment:
```suggestion
    verbose_name = _("Kadaster")
```
it's possible we'll do more things than just API stuff